syntax = "proto3";


service z2ai_text_2_image {
  
  rpc predict(predict_req) returns (predict_resp) {} 

  
}
message predict_req { 
string input1 = 1;
}

message predict_resp { 
string fileName1 = 1;bytes fileBytes1 = 2;
}

